recipe/meta.yaml

```diff
@@ -23,9 +23,12 @@ requirements:
     - {{ compiler('cuda') }}
     - cmake >=3.18
     - make
-  run:
+  host:
     - spdlog =1.7.0
     - thrust >=1.9.0
+  run:
+    # - spdlog =1.7.0  # <--- done via run_exports
```

Comment:
RMM seems to pin this down to the patch release, which that export doesn’t. Not sure if that presents an issue

As RMM is header only we may get some warnings about unused dependencies from conda-build that need to be addressed somehow

Reply:
I checked that on Conda-Forge the latest `spdlog` is 1.8.5, and for the 1.7 series there's only 1.7.0, so we're safe.